### PR TITLE
Expand help on speaker program

### PR DESCRIPTION
**Pull request by MCJack123**
This PR adds some extra detail to the help text for speaker after #1112. Should have added it in that PR, but forgot this file exists.

### CODE REVIEW

File: src/main/resources/data/computercraft/lua/rom/help/speaker.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 The speaker program plays audio files using speakers attached to this computer.
+It supports audio files in either WAV or DFPWM format.
+To convert music into DFPWM, see https://music.madefor.cc.
+The speaker program only supports WAV files using 8-bit PCM or DFPWM.
+WAV files that have more than one channel or a sample rate other than 48000 Hz will not play correctly.
```
Comment (SquidDev) [inline src/main/resources/data/computercraft/lua/rom/help/speaker.md:5]: Sorry, been staring at this for two weeks and just finding the wordng for this incredibly clunky. I'm not really sure how to improve it - this is the best I've got:

```suggestion
It supports audio files in a limited number of formats:
 - DFPWM - You can convert music to DFPWM with external tools like https://music.madefor.cc.
 - WAV - WAV files must be 8-bit PCM or DFPWM, with exactly one channel and a sample rate of 48kHz.
```

I don't know - thoughts?

**Review by SquidDev — CHANGES_REQUESTED** [review]

File: src/main/resources/data/computercraft/lua/rom/help/speaker.md (hunk 1 above)
Comment (Lupus590) [reply src/main/resources/data/computercraft/lua/rom/help/speaker.md:5]: It's an improvement, but it still seems to need something.

Comment (JohnnyIrvin) [reply src/main/resources/data/computercraft/lua/rom/help/speaker.md:5]: What do you think about?

@Lupus590 @SquidDev 
```suggestion
## Audio Formats
- DFPWM
Convert audio to DFPWM with https://music.madefor.cc.

- WAV
8-bit PCM or DFPWM, with one channel and a sample rate of 48kHz.
```

Comment (Lupus590) [reply src/main/resources/data/computercraft/lua/rom/help/speaker.md:5]: @JohnnyIrvin I believe that @SquidDev prefers a good sentence over shorts snippets like this.

Comment (JohnnyIrvin) [reply src/main/resources/data/computercraft/lua/rom/help/speaker.md:5]: Any thoughts on this @SquidDev ?

**Pull request closed without merging**